src/main/groovy/org/shipkit/internal/gradle/GitPlugin.java

```diff
@@ -52,28 +50,18 @@
     static final String WRITE_TOKEN_ENV = "GH_WRITE_TOKEN";
     public static final String GIT_COMMIT_TASK = "gitCommit";
 
-    private final static Logger LOG = Logging.getLogger(GitPlugin.class);
-
     public void apply(final Project project) {
         final ReleaseConfiguration conf = project.getPlugins().apply(ReleaseConfigurationPlugin.class).getConfiguration();
 
         TaskMaker.task(project, GIT_COMMIT_TASK, GitCommitTask.class, new Action<GitCommitTask>() {
             public void execute(final GitCommitTask t) {
                 t.setDescription("Commits all changed files using generic --author and aggregated commit message");
+                //TODO WW create unit tests
                 t.doFirst(new Action<Task>() {
```

Comment:
I assume we use doFirst() so that the configuration can reflect the files to commit, specified by other plugins? Might be worth adding a code comment.